index.js

```diff
@@ -5,46 +5,69 @@ var DEFAULT_FILTER = /^([^\.].*)\.js(on)?$/;
 var DEFAULT_RECURSIVE = true;
 
 module.exports = function requireAll(options) {
-  var dirname = typeof options === 'string' ? options : options.dirname;
-  var excludeDirs = options.excludeDirs === undefined ? DEFAULT_EXCLUDE_DIR : options.excludeDirs;
```

Comment:
I sure noticed that the extra whitespaces generated a lot of noise. Sorry about that. I've tried to remove all of them. Hopefully, I caught all.

How about adding an editor config to ensure anyone gets the desired settings?